lua/dylib.lua

```diff
@@ -53,6 +53,10 @@ struct gk_fib_dump_entry {
 	enum gk_fib_action action;
 };
 
+struct ether_cache {
+	/* This struct has hidden fields. */
```

Comment:
We should have a dump struct like `struct gk_fib_ether_dump_entry` in #89. This will eliminate the need for `l_ether_cache_format()` and give the Lua script the same flexibility already available for FIB entries. My suggestion for this new struct:
```
struct gk_neighbor_dump_entry {

	bool          stale;

	/* The fib action. */
	enum gk_fib_action action;

	/* The IP address of the neighbor. */
	struct ipaddr neigh_ip;

	/* The the MAC address of neigh_ip. */
	struct ether_addr d_addr;
};
```